test/utils/docker-compose.yml

```diff
@@ -0,0 +1,8 @@
+version: "3"
```

Comment:
We have latest version of 3.7 available. Do you think we should use that? 
https://docs.docker.com/compose/compose-file/

Reply:
I am not capable of offering a view on that. But I will find out if there is a specific reason that was not the choice taken by the mosaic team.